utils/certs.go

```diff
@@ -93,7 +93,7 @@ func GenerateCert(hosts []string, certFile, keyFile, caFile, caKeyFile, org stri
 	}
 	// client
 	if len(hosts) == 1 && hosts[0] == "" {
-		template.ExtKeyUsage = []x509.ExtKeyUsage{x509.ExtKeyUsageClientAuth}
+		template.ExtKeyUsage = []x509.ExtKeyUsage{x509.ExtKeyUsageClientAuth, x509.ExtKeyUsageServerAuth}
```

Comment:
Maybe the server keys can be created to do both ClientAuth and ServerAuth, and then to setup a swarm cert people can use the "none" driver, and use those server keys for swarmd?